lib/flex.js

```diff
@@ -53,6 +57,28 @@ class Flex {
       this.sharedSecret = options.sharedSecret;
     }
 
+    function terminate(err) {
+      clearTimeout(gracefulShutdownTimeout);
+
+      if (err) {
+        console.log(`Shutdown Error: ${err}`);
+        process.exit(-1);
+      }
+      process.exit();
+    }
+
+    function shutdown() {
+      if (terminated) {
+        // non-graceful shutdown, requires two successive signals
+        console.log('\nForced quit!\n');
```

Comment:
SDK `console.log` messages will appear in user logs (`kinvey flex logs`). These should probably be removed to be consistent with the rest of the codebase